Module3_Functional_Programming/src/pureFunctions.js

```diff
@@ -0,0 +1,21 @@
+function getPrivateAndWorkTasks(tasksArr) {
+    return [
+            ...concatSameTasks(tasksArr.filter(({type}) => type === "work" || type === "private"))
+            .values()
+        ]
+        .map(({name, duration}) => {
+            return `Total duration of '${name}' is ${duration}`;
```

Comment:
looks good)
minor: in the answer should be hours instead of minutes)

Reply:
fixed